### ISSUE TITLE
Split Object/Face Recognition

### ISSUE THREAD

**Issue by tknobi** [meichthys/foss_photo_libraries#16]
Hi,

For me, recognizing objects and recognizing faces are two completely different things. While objects are simply recognized, faces have to be detected, a person has to be recognized, suggested, labeled, filtered, clustered, ....

So should this be split?

Cheers
tknobi

**Comment by meichthys** [comment]
Yes, i agree. I would think that any application that has face detection will also have object detection, but the quality of object detection could still vary widely.

We should probably also split out some other 'Features' like Mobile App and Desktop App. I'll create separate issues for those.
